notes-webapp/src/main/webapp/vue-app/notes-editor/components/NotesEditorDashboard.vue

```diff
@@ -618,6 +618,7 @@ export default {
           },
           change: function (evt) {
             self.note.content = evt.editor.getData();
+            self.autoSave();
```

Comment:
I think this is not necessary.
There is a watch on note.content, which call autosave().
Can you check

Reply:
we need to keep it.